CareWheels/www/app/views/individualStatus.html

```diff
@@ -38,9 +45,9 @@ <h2 class="title">Sensor Status, Trevor</h2>
     </div>
     <div class="row">
```

Comment:
it might be worth while to implement a ng-repeat here, to fill in each row dynamically